internal/cri/server/podsandbox/controller_test.go

```diff
@@ -45,9 +45,10 @@ var testConfig = criconfig.Config{
 // newControllerService creates a fake criService for test.
 func newControllerService() *Controller {
 	return &Controller{
-		config: testConfig,
-		os:     ostesting.NewFakeOS(),
-		store:  NewStore(),
+		config:    testConfig,
+		os:        ostesting.NewFakeOS(),
+		osManager: ostesting.NewFakeOSManager(),
```

Comment:
I think this "fake" type shouldn't be necessary if the `statManager` wraps the `OS` interface (which could be either a real or fake implementation):
```go
os := ostesting.NewFakeOS()
return &Controller{
  ...
  os: os,
  // The same type you would use in the actual controller.
  osManager: osinterface.NewStatManager(os),
}
```